### PR TITLE
Portfolio: finish mobile version :fast_forward:

### PR DESCRIPTION
**Pull request by HAMMAS-SALEEM**
### **Things added:** :memo:

:heavy_check_mark: Work & About-Me Sections in Markup (HTML)
:heavy_check_mark: Style Work Section (CSS)
:heavy_check_mark: Some Debugging of HTML & CSS
:heavy_check_mark: Followed GitHub Flow
:heavy_check_mark: Followed Linters
:heavy_check_mark: Documented work in a professional way

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,19 +30,127 @@
...
+                </p>
+                <div class="bcontain">
+                    <button class="w1btn" disabled>html</button>
+                    <button class="w1btn w1b" disabled>css</button>
+                    <button class="w1btn w1b" disabled>javaScript</button>
+                </div>
+                <button class="w1btn2">See Project</button>
+            </div>
+        </div>
+        <div class="about-me">
```
Comment (ignatius22) [inline index.html:132]: - [ ] Let's make sure we follow the proper guidelines by making this container a` section` tag

File: index.html [hunk 2]
```diff
@@ -30,19 +30,127 @@
...
+                    <ul class="icon-list">
+                        <li><a href="#" class="fa fa-twitter fa-color" aria-hidden="true"></a></li>
+                        <li><a href="#" class="fa fa-linkedin fa-color" aria-hidden="true"></a></li>
+                        <li><a href="#" class="fa fa-medium fa-color" aria-hidden="true"></a></li>
+                        <li><a href="#" class="fa fa-github fa-color" aria-hidden="true"></a></li>
+                        <li> <a href="#" class="fa fa-angellist fa-color" aria-hidden="true"></a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="cr-grid">
```
Comment (ignatius22) [inline index.html]: - [ ] Let's ensure we follow good practice by using more semantic tags  (`section`  tag) for this container.

File: index.html [hunk 3]
```diff
@@ -30,19 +30,127 @@
...
+                <div class="bcontain">
+                    <button class="w1btn" disabled>html</button>
+                    <button class="w1btn w1b" disabled>css</button>
+                    <button class="w1btn w1b" disabled>javaScript</button>
+                </div>
+                <button class="w1btn2">See Project</button>
+            </div>
+            <div class="w1contain">
+                <img class="pti" src="./images/SP2.png" alt="Portfolio Title Image">
+                <h2 class="works-title">Multi-Post Stories</h2>
```
Comment (ignatius22) [inline index.html]: - [ ] I think it would be nice to use the `header` tag for our project title.

File: index.html [hunk 4]
```diff
@@ -30,19 +30,127 @@
...
+                        <li><a href="#" class="fa fa-medium fa-color" aria-hidden="true"></a></li>
+                        <li><a href="#" class="fa fa-github fa-color" aria-hidden="true"></a></li>
+                        <li> <a href="#" class="fa fa-angellist fa-color" aria-hidden="true"></a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="cr-grid">
+            <div class="w1contain">
+                <img class="pti" src="./images/SP.png" alt="Portfolio Title Image">
+                <h2 class="works-title">Tonic</h2>
```
Comment (ignatius22) [inline index.html]: - [ ] I think it would be nice to use the `header` tag for our project title.

**Review by ignatius22 — CHANGES_REQUESTED** [review]
## Hi @HAMMAS-SALEEM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Having a descriptive class name is a very good practice that can make you more professional


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HAMMAS-SALEEM** [comment]
Thanks for reaching me out. I want to ask about the last requirement. I did
not know what project title are you talking about. Can you tell me in which
section and for which project title i need to use header tag.

**Comment by HAMMAS-SALEEM** [comment]
One more thing I want to talk about is should I use one section tag for
about-me because ul and li are inside this tag so why should I use another
section for li such as Languages, Frameworks, skills.

**Comment by HAMMAS-SALEEM** [comment]
I wanted to ask about work section not about-me section
Sorry for my mistake.

**Review by Cathella — APPROVED** [review]
## STATUS: APPROVED 🟢

### Hi @HAMMAS-SALEEM 👋🏻, 

You have done a tremedous job working on the changes requested by the previous reviewer 💪🏻. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**